src/features/boards/article/ArticleLikeButton.tsx

```diff
@@ -0,0 +1,54 @@
+import Icon from "@/components/icon";
+import { useResponsive } from "@/hooks/use-responsive";
+
+export default function ArticleLikeButton({
+  likeCount,
+  isLiked,
+  onToggle,
+}: {
+  likeCount?: number;
+  isLiked: boolean;
+  onToggle: () => void;
+}) {
+  const { isDesktop, isMobile } = useResponsive();
+
+  return (
+    <>
+      {isDesktop ? (
+        <div className="h-10">
+          <div className="absolute top-44 -right-20 flex flex-col items-center gap-2">
+            <button
+              onClick={onToggle}
+              className="flex h-16 w-16 cursor-pointer items-center justify-center rounded-full bg-background-inverse"
+            >
+              <Icon name={isLiked ? "heartFill" : "heart"} color="white" />
+            </button>
+            <span className="text-lg-r text-state-400">{likeCount}</span>
+          </div>
+        </div>
+      ) : (
+        <span className="flex h-[49px] items-center justify-end text-md-r text-slate-400 tablet:h-20 tablet:text-lg-r">
+          <button
+            onClick={onToggle}
+            className="mr-1 inline-block cursor-pointer"
+          >
+            {isMobile ? (
+              <Icon
+                name={isLiked ? "heartFill" : "heart"}
+                size="small"
+                color="white"
+              />
+            ) : (
+              <Icon
+                name={isLiked ? "heartFill" : "heart"}
+                size="large"
+                color="white"
+              />
```

Comment:
Icon 전체를 분기하지 않고도 size만 조건부로 주면 중복이 조금 줄어들 수 있을 것 같습니다.
```suggestion
              <Icon
                name={isLiked ? "heartFill" : "heart"}
                size={isMobile ? "small" : "large"}
                color="white"
              />
```

Reply:
더 간결하게 가능했었네요!🤣 감사합니다!! 수정해서 반영하였습니다!